Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -55,140 +59,135 @@ class TransactionHistoryViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         if !AppDelegate.walletLoader.isSynced {
-            print(" wallet not synced on history")
+            print("wallet not synced on history")
             return
         }
-        self.visible = true
-        if (self.FromMenu){
-            refreshControl.showLoader(in: self.tableView)
-            loadTransactions()
-            FromMenu = true
-        }
+        self.refreshControl.showLoader(in: self.tableView)
+        loadTransactions()
     }
     
-    override func viewDidDisappear(_ animated: Bool) {
-        super.viewDidDisappear(animated)
-        self.visible = false
-        dismiss(animated: true, completion: nil)
+    func initFilterBtn() {
+        self.btnFilter.initMenu(self.filterMenu) { [weak self] index, value in
+            self?.applyTxFilter(currentFilter: DcrlibwalletTxFilterAll)
```

Comment:
`index` and `value` indicate the currently selected filter item, this line should apply the current/changed filter selection rather than always filter all transactions.

Reply:
changed in 9a8c24b